### ISSUE TITLE
#TinyBert Training Pipeline Problems

### ISSUE THREAD

**Issue by mexiQQ** [huawei-noah/Pretrained-Language-Model#153]
Hi Huawei team:

Sorry to disturb you, can you guys answer my following question?

Why did the training pipeline of TinyBert "general_distill.py" not use DDP to initialize the student model, instead of only initializing the teacher model? And why there is no synchronization of the normalization layer?

![image](https://user-images.githubusercontent.com/6386781/140790757-009842b4-711b-4060-b72a-93123c336360.png)

And when opening the mixed-precision,  where can I find the function "backward" from "optimizer"？

![image](https://user-images.githubusercontent.com/6386781/140790709-f5dd8363-ec05-4125-9e8a-f57ac0e10949.png)

thx

**Comment by zwjyyc** [comment]
hi mexiQQ,
This code does not support fp16 training and DDP training. So the relevant part is redundant.
Please refer to the AutoTinyBERT code which supports both fp16 and DDP training.

**Comment by mexiQQ** [comment]
Thanks for your reply.
